Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/IdentityProvidingClients/IdentityProvidingSTSClient.swift

```diff
@@ -5,6 +5,8 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Added function to `IdentityProvidingSTSClient` protocol for STS assume role cred resolver. The struct that conforms to this protocol is code-generated under InternalAWSSTS. It creates an instance of the internal STS client, takes argument values to cal assumeRole, then returns `AWSCredentialIdentity` object constructed from response values.